sqlitecluster/SQLiteNode.cpp

```diff
@@ -1028,7 +1026,7 @@ bool SQLiteNode::update(uint64_t& nextActivity) {
         SASSERTWARN(!_masterPeer);
 
         // Are we waiting for approval of a distributed transaction?
-        if (_currentCommand && !_currentCommand->transaction.empty()) {
+        if (_currentCommand) {
```

Comment:
It's impossible to set `_currentCommand` without setting it's `transaction` property. There's only one place where we set `_currentCommand` to anything besides null, and it always sets transaction properties. Removing this extra condition makes it much easier to reason about what happens after this block.